tests/cli/info/owners/test_code.py

```diff
@@ -0,0 +1,186 @@
+# SPDX-FileCopyrightText: 2025-present Datadog, Inc. <dev@datadoghq.com>
+#
+# SPDX-License-Identifier: MIT
+from __future__ import annotations
+
+import json
+from os import sep
+from typing import TYPE_CHECKING
+
+from dda.utils.fs import Path
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable
+
+DEFAULT_CODEOWNERS_LOCATION = Path(".github/CODEOWNERS")
```

Comment:
Let's make this a fixture to avoid logic occurring upon import like so:

https://github.com/DataDog/datadog-agent-dev/blob/f8026d5d3566f06f719a317d64b04c2c370e11b2/tests/env/dev/types/test_linux_container.py#L30-L32